public/app/core/components/RolePicker/RolePickerMenu.tsx

```diff
@@ -37,9 +37,20 @@ const fixedRoleGroupNames: Record<string, string> = {
 };
 
 const noBasicRoleFlag = contextSrv.licensedAccessControlEnabled() && config.featureToggles.noBasicRole;
-const tooltipMessage = noBasicRoleFlag
-  ? 'You can now select the "No basic role" option and add permissions to your custom needs.'
-  : undefined;
+const tooltipMessage = noBasicRoleFlag ? (
+  <>
+    You can now select the &quot;No basic role&quot; option and add permissions to your custom needs. You can find more
+    information in&nbsp;
+    <TextLink
+      href="https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/#organization-roles"
+      variant="bodySmall"
+      external
+    >
+      our documentation
+    </TextLink>
+    .
+  </>
+) : undefined;
```

Comment:
for both checks for `noBasicRoleFlag` i think we have to have a empty string instead of undefined reading the <Tooltip> content field.

https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Tooltip/Tooltip.tsx#L16



defintion here - https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Tooltip/types.ts#L10

Reply:
This is intentional. If we return an empty string, the tooltip will be displayed with an empty message. By using `undefined` the content of the tooltip won't be rendered since it is controlled [here](https://github.com/grafana/grafana/blob/cd744eaec404c5fbdcdc8dd3dedfe0ded2747f19/public/app/core/components/RolePicker/BuiltinRoleSelector.tsx#L38).

Reply:
I see that it is being controlled there. It's a nit really, in my opinion we should send in the correct types to each interface in the code (as a good practice, too nitty perhaps).

The check for the (`{tooltipMessage &&`) will also be caught with empty string`""`.

You decide on the way forward here ⚡

Reply:
Changed and merged